staging/src/k8s.io/component-base/tracing/api/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-instrumentation-approvers
```

Comment:
replace this bit with this:

```
options:
  no_parent_owners: true
approvers:
  - api-approvers
```

to make sure changes route through API approval since changes here can impact compatibility of component config files

Reply:
updated